### ISSUE TITLE
Suggestion: crop video

### ISSUE THREAD

**Issue by mauron85** [vvo/gifify#56]
Hi, outstanding work. Is it possible to add video cropping?

**Comment by vvo** [comment]
Yes, taking a PR to do so, if ffmpeg has it then we can do it.

**Comment by mauron85** [comment]
yeah. ffmpeg does have it https://ffmpeg.org/ffmpeg-filters.html#crop so should be possible

**Comment by vvo** [comment]
Ok, have a look at the code, you might be surprised how easy it can be to add

**Comment by mauron85** [comment]
Definitely. :-) So is it now so, that you will implement it?

**Comment by vvo** [comment]
Sorry no I will not implement it, but I am really interested in it and I invite you to submit a pull request or having a try at it.

Clone the repo, type npm link in it (it will make gifify available globally), hack the code, try some videos, push a pull request!

**Comment by mauron85** [comment]
I feel with you. From my own experience, I know how hard (impossible) is to convince people to do PR. I'll probably take look on it, when I find spare time between my biz and another [open source project](https://github.com/mauron85/cordova-plugin-background-geolocation) I'm working on.
